### PR TITLE
[Bugfix] lizmap_search application context from project

### PR DESCRIPTION
**Pull request by rldhont**
The lizmap_search can be filtered by user.

* Funded by 3liz

### CODE REVIEW

File: lizmap/modules/lizmap/controllers/searchFts.classic.php [hunk 1]
```diff
@@ -30,12 +30,58 @@ public function get()
...
+
+            return $rep;
+        }
+
         $pquery = filter_var($pquery, FILTER_SANITIZE_STRING, FILTER_FLAG_NO_ENCODE_QUOTES);
 
         // Run query
         $fts = jClasses::getService('lizmap~lizmapFts');
-        $data = $fts->getData($project, $pquery);
+        jLog::log('The lizmap project '.strtoupper($lproj->getKey()).' exists!', 'errors');
```
Comment (Gustry) [inline lizmap/modules/lizmap/controllers/searchFts.classic.php:83]: error ? while the project exist ?
Maybe remove this log.

File: tests/qgis-projects/tests/set_tests_lizmap_search.sql [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+CREATE EXTENSION IF NOT EXISTS unaccent WITH SCHEMA public;
+
+-- Add the f_unaccent function to be used in the index
+CREATE OR REPLACE FUNCTION public.f_unaccent(text)
+RETURNS text AS
+$func$
+SELECT public.unaccent('public.unaccent', $1)  -- schema-qualify function and dictionary
+$func$ LANGUAGE sql IMMUTABLE;
+
+CREATE SCHEMA IF NOT EXISTS lizmap;
```
Comment (Gustry) [inline tests/qgis-projects/tests/set_tests_lizmap_search.sql:14]: the `search_path` is done to `lizmap` already before ?

File: lizmap/modules/lizmap/controllers/searchFts.classic.php (hunk 1 above)
Comment (laurentj) [inline lizmap/modules/lizmap/controllers/searchFts.classic.php:83]: I guess this line was for debugging, it should be removed

**Review by laurentj — APPROVED** [review]
lgtm

File: lizmap/modules/lizmap/controllers/searchFts.classic.php (hunk 1 above)
Comment (laurentj) [reply lizmap/modules/lizmap/controllers/searchFts.classic.php:83]: it's ok for me if this line is removed

Comment (rldhont) [reply lizmap/modules/lizmap/controllers/searchFts.classic.php:83]: it was for testing, something wrong in the docker refreshing files

**Comment by rldhont** [comment]
I have reduced phpstan errors and fix loading SQL files for cypress tests.

**Pull request merged**